assets/js/ajax-filters.js

```diff
@@ -239,14 +239,11 @@ jQuery( document ).ready( function ( $ ) {
 		return false;
 	} );
 
-	if ( $.isFunction( $.fn.select2 ) ) {
-		var select2_args = {
-			allowClear: true,
-			minimumResultsForSearch: 10
-		};
-		if ( 1 === parseInt( job_manager_ajax_filters.is_rtl, 10 ) ) {
```

Comment:
Just wondering is this was intentionally removed?

Reply:
That is now coming in from `job_manager_select2_args`.